spec/person_spec.rb

```diff
@@ -0,0 +1,21 @@
+require './person'
+
+person = Person.new(30, 'Addis')
+
+describe '#Review Person Class' do
+  it 'Returns Name' do
+    expect(person.name).to eql 'Addis'
+  end
+
+  it 'Returns Age' do
+    expect(person.age).to eql 30
+  end
+
+  it 'Return result for can_use_services?' do
+    expect(person.can_use_services?).to eql true
+  end
+
+  it 'Return result for correct_name' do
+    expect(person.correct_name).to eql 'Addis'
+  end
+end
```

Comment:
- [ ] Great job so far 👏🏻 👏🏻 However, according to the project requirements, there should be unit tests for all the methods in all the classes, Kindly make sure the `add_rentals`  method of the class person has (at least) an 'it' block 👇🏻 check your person.rb file

```ruby
  def add_rentals(rental)
     @rentals.push(rental)
     rentals.person = self
  end
```

Reply:
Thank you for the feedback. Test for add_rental has been implemented.